english/timeTunnel/timeTunnel.md

```diff
@@ -2,4 +2,14 @@ Upon exiting the time tunnel, it is clear you have entered the future.
 There are flying cars complete the the whirring noise.
 However, marshmellows have been banned.
 This must be corrected, but how?
-Suddenly, a cat walks up and says "Is that a marshmallow on your fork?"
\ No newline at end of file
+Suddenly, a cat walks up and says "Is that a marshmallow on your fork?"
+
+You quickly stuff the marshmellow in your mouth and garble out a quick, "no".
+
+The cat suspiciously notes, "alrighty then, nyaan. Good thing it wasn't, otherwise, I would've had to take you away, nyaan. I'll be off now!"
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.